components/developer/gdb/manifests/sample-manifest.p5m

```diff
@@ -10,10 +10,11 @@
 #
 
 #
-# Copyright 2021 <contributor>
+# Copyright 2022 <contributor>
 #
 
 set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
```

Comment:
`pkg.human-version` should be added to `gdb.p5m` too.

Reply:
Done